tests/test_graft.py

```diff
@@ -95,8 +95,9 @@ def test_graft_axon_on_synthesized_cell():
     graft.graft_axon(synthesized_cell, donor_neuron)
     axon = graft.find_axon(synthesized_cell)
     assert_array_almost_equal(axon.points,
-                              [[5.1272364, 5.4825425, 4.9689593],
-                               [5.1272364, 1.4825425, 4.9689593]])
+                              [[1.669128, 8.830159, 4.592799],
```

Comment:
this is the only change that I'm uncomfortable with; I'm not sure why the expectation was correct, or whether this is correct.

Reply:
@eleftherioszisis; can you have a look at this?

Reply:
As discussed this was expected because the soma radius was different.